README.md

```diff
@@ -24,7 +24,7 @@ Hydro provides a high-level language for the majority of developers called [Hydr
 
 ## Development Setup
 
-See the [quickstart section of the Hydroflow+ book](https://hydro.run/docs/hydroflow_plus/quickstart/) for instructions on installing Rust and getting started with the Hydroflow+ template.
+See the [quickstart section of the Hydroflow+ book](https://hydro.run/docs/hydro/quickstart/) for instructions on installing Rust and getting started with the Hydroflow+ template.
```

Comment:
Needs name change edits. I have a separate PR #1621 with suggested wording, but you'll still need to change the names of directories/files in the docs and links to those in the version in my PR.

Reply:
Yep. I will merge #1621 last. 

Thanks for putting that CR together!